packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js

```diff
@@ -406,7 +407,11 @@ export const getRoutes = (locals) => {
     if (typeof routes === 'function') {
         _routes = routes()
     }
-    const allRoutes = [..._routes, {path: '*', component: Throw404}]
+    const allRoutes = [
+        {path: '/__storefront-preview/refetch-data', component: RefetchData},
```

Comment:
Good point. The reason it's like that because `_routes` from a project can have a fallback route (with `path: '*'`) that will inadvertently match the `__storefront-preview` route. I think it's safe that our route comes first and to assume that customer projects won't have a route like that.